src/megaapi_impl.cpp

```diff
@@ -6371,12 +6371,34 @@ bool MegaApiImpl::createPreview(const char *imagePath, const char *dstPath)
     fsAccess->path2local(&utf8DstPath, &localDstPath);
 
     sdkMutex.lock();
-    bool result = gfxAccess->savefa(&localImagePath, GfxProc::PREVIEW1000x1000, &localDstPath);
+    bool result = gfxAccess->savefa(&localImagePath, 1000, 1000, &localDstPath);
```

Comment:
It would be better to use `GfxProc::dimensions[GfxProc::PREVIEW1000x1000][0]` and `GfxProc::dimensions[GfxProc::PREVIEW1000x1000][1]` instead of hardcoded dimensions. Or maybe adapt the size declarations to something more usable with the new prototype of `savefa()`